front/src/modules/timesStops/utils.ts

```diff
@@ -43,3 +47,36 @@ export const formatSuggestedViasToRowVias = (
     };
   });
 };
+
+export function findNextScheduledPoint(
+  operationalPoints: OperationalPointWithTimeAndSpeed[],
+  tsPathPointsWithOpPointIndices: PathPointOpPointCorrespondance[],
+  sugOpIndex: number
+) {
+  const pathIndex = tsPathPointsWithOpPointIndices.findIndex(
+    (pathPoint) => pathPoint.correspondingOpPointIndex === sugOpIndex
+  );
+  const nextPathPoint = tsPathPointsWithOpPointIndices[pathIndex + 1];
+  return operationalPoints[nextPathPoint?.correspondingOpPointIndex];
+}
+
+const getDigits = (u: string | undefined) => (u === 's' || u === '%' ? 0 : 1);
```

Comment:
Maybe getDigitsNumber is clearer. Also, you can rename u `unit`
```suggestion
const getDigitsNumber = (unit: string | undefined) => (unit === 's' || unit === '%' ? 0 : 1);
```

Reply:
done